experiments/yarn/src/main/scala/org/apache/gearpump/experiments/yarn/AppConfig.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gearpump.experiments.yarn
+
+import org.apache.gearpump.cluster.main.{ParseResult, ArgumentsParser, CLIOption}
+import com.typesafe.config.Config
+
+object EnvVars {  
```

Comment:
Is EnvVars  specifically used in class AppConfig or also used else where?

If this is also used else where, I will recommend to store the EnvVars to a stand alone with a more specific name like YarnConstants